tests/test_searchspace.py

```diff
@@ -86,10 +87,10 @@ def test_discrete_searchspace_creation_from_dataframe():
 
 def test_continuous_searchspace_creation_from_bounds():
     """A purely continuous search space is created from example bounds."""
-    parameters = [
+    parameters = (
```

Comment:
one question coming form the following perspective:
when using baybe objects I much rahter put collections ob objects in asa a list. Maybe thats just prefrence, but it has one advantage in that I dont need the trailing comma to define tuples in case the sequence has only one entry like: `(1,)` (just had a case where someone forgot that causing an issue)

So: is it still supported to provide eg the parameter sequences (eg for searchspace) as list? Will this raise a highlighted typing warning in the IDE? If so I think its seriously worth considering not typing the sequences as tuples but lists, any particular reasons for why we need them to be tuples and not lists?

Reply:
perfect, if its an external issue we should not be bothered by that and implementation seems fine to me